### PR TITLE
Updating grid picker and its API. 

### LINKED ISSUE #1527: Can no longer scroll canvas with mouse wheel
Repros on live website

### PR DESCRIPTION
**Pull request by microsoftsam**
Updating grid picker and its API. 

screen shot 2017-03-03 at 2 40 59 pm

Also enabling Blockly mouse wheel, as we've ported in the change to scroll / zoom. Fixes #1527
Updating Blockly.

### CODE REVIEW

File: pxtlib/emitter/emitter.ts [hunk 1]
```diff
@@ -393,6 +393,7 @@ namespace ts.pxtc {
     export function parseCommentString(cmt: string): CommentAttrs {
         let res: CommentAttrs = {
             paramDefl: {},
+            blockFieldEditorParams: {},
```
Comment (pelikhan) [inline pxtlib/emitter/emitter.ts]: make it lazy

File: pxtlib/emitter/emitter.ts [hunk 2]
```diff
@@ -405,6 +406,8 @@ namespace ts.pxtc {
                     let v = v0 ? JSON.parse(v0) : (d0 ? (v0 || v1 || v2) : "true");
                     if (U.endsWith(n, ".defl")) {
                         res.paramDefl[n.slice(0, n.length - 5)] = v
+                    } else if (U.startsWith(n, "blockFieldEditorParams")) {
+                        res.blockFieldEditorParams[n.slice(23, n.length)] = v
```
Comment (pelikhan) [inline pxtlib/emitter/emitter.ts]: lazy init here

File: webapp/src/blocks.tsx [hunk 3]
```diff
@@ -546,7 +546,7 @@ export class Editor extends srceditor.Editor {
             zoom: {
                 enabled: false,
                 controls: false,
-                /* wheel: true, wheel as a zoom is confusing and incosistent with monaco */
+                wheel: true,
```
Comment (pelikhan) [inline webapp/src/blocks.tsx]: this change seems unrelated

Comment (microsoftsam) [inline webapp/src/blocks.tsx]: unrelated. makes merging easier, blockly dependent.
